hw/ip/dma/data/dma.hjson

```diff
@@ -41,6 +41,33 @@
       fields: [
         { bits: "31:0", name: "DONE", desc: "Copy done" }
       ]
+    },
+    { name:     "SRC_PTR_INC",
+      desc:     "Increment number of source pointer every time a word is copied from source",
+      swaccess: "rw",
+      hwaccess: "hro",
+      resval:   4,
+      fields: [
+        { bits: "31:0", name: "SRC_PTR_INC", desc: "Source pointer increment" }
+      ]
+    },
+    { name:     "DST_PTR_INC",
+      desc:     "Increment number of source pointer every time a word is copied to destination",
+      swaccess: "rw",
+      hwaccess: "hro",
+      resval:   4,
+      fields: [
+        { bits: "31:0", name: "DST_PTR_INC", desc: "Destination pointer increment" }
+      ]
+    },
```

Comment:
When using the SPI we want to set this INC parameter to zero since we are popping the data from a FIFO. When implementing this I was discussing it with Davide and he told me to set these configuration parameters to have more flexibility from SW